static-assets/components/cstudio-forms/data-sources/embedded-content.js

```diff
@@ -39,6 +40,7 @@ YAHOO.extend(CStudioForms.Datasources.EmbeddedContent, CStudioForms.CStudioFormD
   itemsAreContentReferences: true,
 
   createElementAction: function (control, _self) {
+    _self.inserted = false;
```

Comment:
Should this be cleared at some point or is the instantiation lifecycle enough to manage that state?